mantidproject/mantid
# Implement asymmetric Pearson-VII fit function

**Description of work.**

This PR contains an implementation of the asymmetric Pearson VII fit function, which describes peak shapes from conventional X-ray powder diffraction patterns. 

It resolves the issue #14617 



**To test:**

1. Proofread documentation, check if all formulas are correct
2. Check if unit tests make sense
3. Examine the function's behavior for known limiting cases



Fixes #14617 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
